organization/summit.md

```diff
@@ -0,0 +1,23 @@
+# Mentor Summit
+
+Every year Google promote a Summit
+and each [**mentoring**] organization,
+**in this case only NumFOCUS**,
+may send two (2) members to the Mentor Summit.
+
+NumFOCUS will nominate as Summit ambassador
+the lead administrator of the current year
+**and** the lead administrator of following year.
```

Comment:
@kain88-de I agree with you. @aterrel and I hope that lead administrator stay for more than one edition so this rules only are used a few times. Reword suggestions are welcome.